app/components/certificate/description.tsx

```diff
@@ -38,6 +45,20 @@ export default function DescriptionSection({
                 <Text fontWeight="bold">Expires On</Text>
                 <Text>{validToFormatted}</Text>
               </VStack>
+
+              <Flex justifyContent="flex-end">
+                <Form method="post">
+                  <IconButton
+                    type="submit"
+                    aria-label="renew-certificate"
+                    icon={<RepeatIcon />}
+                    backgroundColor="transparent"
+                    color="black"
+                    _hover={{ backgroundColor: 'brand.500', color: 'white' }}
+                    isDisabled={isRenewable ? false : true}
```

Comment:
`isDisabled={!isRenewable}`

Reply:
I think I just didn't want to confuse myself so I wrote it like that, I'll change it now